src/app/core/services/fss-search-filter.service.ts

```diff
@@ -77,4 +77,20 @@ export class FssSearchFilterService {
     return operatorType
   }
 
+  getFilterExpressionForSimplifiedSearch(fssSearchFilter: string) {
```

Comment:
Please specify the return type
```suggestion
  getFilterExpressionForSimplifiedSearch(fssSearchFilter: string): string {
```

Reply:
Resolved